### DISCUSSION TITLE
Cannot open Firefly Wallet after update...

### DISCUSSION THREAD

**Discussion by Denyel111** [iotaledger/firefly#7406] [Help & Support]
App Version: 1.7.4

platform: macOS
platformVersion: 10.12.6
platformArchitecture: x64
cpuCount: 4
totalMem: 16384.0 MB
freeMem: 4985.4 MB

Error Type: [Preload Context] Error

Error: node-loader:
Error: dlopen(/Applications/Firefly.app/Contents/Resources/app.asar.unpacked/public/build/index.node, 1): Symbol not found: __ZdlPvSt11align_val_t
  Referenced from: /Applications/Firefly.app/Contents/Resources/app.asar.unpacked/public/build/index.node
  Expected in: /usr/lib/libc++.1.dylib
 in /Applications/Firefly.app/Contents/Resources/app.asar.unpacked/public/build/index.node

**Comment by begonaalvarezd** [comment]
Hi, unfortunately we dont provide support anymore for macOS versions <= 10.13 in Firefly IOTA. We recommend upgrading to a newer version, at least 10.15, higher if possible so you can get OS security updates

**Reply by Denyel111** [reply]
Ok, i understand... unfortunaly my mac is a 2013 model and there is no longer support. Is there a possibility that i can download a older firefly version?

**Reply by HankMoody089** [reply]
As Begona said, no, the older versions of MacOS are not supported any more, and that for good reason. 
You do not get security updates of MacOS any more, that are very crucial when doing crypto things. Time to get a newer Mac that supports latests MacOS.

**Reply by Denyel111** [reply]
I understand, but i just want to get my iota, smr and asmb for the moment... before the update everything was fine. And i want to make sure that all the staking wasn´t for nothing...

**Reply by Denyel111** [reply]
Is there a possibility to get a old firefly version that fits my old mac ?

**Reply by MiKeF05** [reply]
You can find all released versions here: https://github.com/iotaledger/firefly/releases

**Reply by Denyel111** [reply]
thanks! version 1.7.2 still works for me... i will looking for a newer Mac soon ;)
